.github/workflows/tooljet-docker-latest-build.yml

```diff
@@ -0,0 +1,49 @@
+name: Tooljet-lastest Docker Image
```

Comment:
latest*

Reply:
actually latest doesn't need a separate workflow, we could tag the latest within the release workflow itself